magicctapipe/scripts/lst1_magic/semi_automatic_scripts/check_MAGIC_runs.py

```diff
@@ -224,8 +212,9 @@ def main():
     )
 
     tel_id = [1, 2]
-
-    database = table_magic_runs(df, args.date_min, args.date_max)
+    date_min = args.begin if args.begin != 0 else date_min_default
```

Comment:
why do you use .begin .end instead of accessing it by the parameter name? it looks quite unsafe this way

Reply:
sorry, my mistake, I did not realize that the names were now "begin" and "end", resolving comment